packages/react-components/react-utilities/src/hooks/useOnScrollOutside.ts

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react';
 import { useEventCallback } from './useEventCallback';
-import type { UseOnClickOrScrollOutsideOptions } from './useOnClickOutside';
+import { UseOnClickOutsideOptions } from './useOnClickOutside';
+
+export type UseOnScrollOutsideOptions = Pick<UseOnClickOutsideOptions, 'element' | 'refs' | 'contains' | 'disabled'> & {
+  /**
+   * Called if the scroll is outside the element refs
+   */
+  callback: (ev: Event | MouseEvent | TouchEvent) => void;
+};
```

Comment:
Should this have `/** @internal */` since it appears to only be used by an internal utility?

Reply:
We could add it, but I'm not sure what the use of `@internal` is anymore, even if these utilties are internal we are still bound by the requirement to never break them, now that our API surface includes our entire set of packages 🥲